# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/loader/ScriptCacheActors.cpp
# Commit: 2bc5949902ac
# Full Hash: 2bc5949902acc042205fc7e85260bbbc37dbedc7
# Author: Matthew Gaudet <mgaudet@mozilla.com>
# Date: 2021-09-18 09:46:57
# Regressor Bug: 1688788
# File Overlap Count: 1
# Description:
#   Bug 1688788 - Convert ScriptPreloader to manage Stencils, not Scripts. r=kmag,tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113013
# ==============================================================================

diff -r e660e2c3c3d1 -r 2bc5949902ac js/xpconnect/loader/ScriptCacheActors.cpp
--- a/js/xpconnect/loader/ScriptCacheActors.cpp	Sat Sep 18 01:29:45 2021 +0000
+++ b/js/xpconnect/loader/ScriptCacheActors.cpp	Sat Sep 18 01:29:45 2021 +0000
@@ -79,8 +79,8 @@
 
   auto& cache = ScriptPreloader::GetChildSingleton();
   for (auto& script : scripts) {
-    cache.NoteScript(script.url(), script.cachePath(), processType,
-                     std::move(script.xdrData()), script.loadTime());
+    cache.NoteStencil(script.url(), script.cachePath(), processType,
+                      std::move(script.xdrData()), script.loadTime());
   }
 
   return IPC_OK();